os/android/drmhwctwo.h

```diff
@@ -239,6 +239,9 @@ class DrmHwcTwo : public hwc2_device_t {
   hwcomposer::GpuDevice device_;
   std::map<hwc2_display_t, HwcDisplay> displays_;
   std::map<HWC2::Callback, HwcCallback> callbacks_;
+
+ private:
+  static bool is_explicit_sync_enabled_;
```

Comment:
what we could do is pass the state to display, display queue whom should be already creaed at this point. Once displayqueue starts to initialize compositor etc it can set that info appropriately